default-parameters-es6/demo.js

```diff
@@ -0,0 +1,41 @@
+// Supply default values for parameters
+
+// Before, using ES5...
+function greetES5(message, to) {
+  // Test parameter values in the function body, specifying a default
+  to = to || 'DOM';
+  // OR if (typeof to === undefined) { to = 'DOM'; }
+  // OR if (arguments.to === 1) { to = DOM'; '}
```

Comment:
I don't understand `arguments.to` in this context. Do you mean `arguments.length`?

Also, your `'` character in `to = DOM'; '` seems to have floated off to the wrong position.